website/content/docs/enterprise/audit-logging.mdx

```diff
@@ -8,9 +8,11 @@ description: >-
 # Audit Logging
 
 <EnterpriseAlert>
-  This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Governance and Policy module.
+  This feature is available in v1.8.0+ of: (1) {' '}
```

Comment:
**Question for reviewer**

Do we agree with including the version inline here? (versus just `is available in: (1) ...`

And should it be `v1.8.0+` or `versions 1.8.0+`?